docs/src/shutdown.md

```diff
@@ -0,0 +1,16 @@
+# Shutdown
+
+It is not possible to explicitly close connection to the database.
+Given connection is closed, when the client varriable assosiated with this connection,
+is collected by the garbage collector.
```

Comment:
:question: Isn't this self-contradictory? If only Node JS supports explicit deallocation...

Ah, I understood. Even though removal of references to the object is explicit, the actual GC run that frees objects that have no active references to is implicit. Maybe let's add this explanation to the docs?